apps/gro/steps.js

```diff
@@ -126,13 +126,54 @@ module.exports = {
     template: 'email',
     fields: ['email-text'],
     continueOnEdit: true,
-    next: '/post'
+    next: '/country'
   },
-  '/post': {
-    controller: require('./controllers/post'),
-    template: 'post',
+  '/country': {
+    fields: [
+      'country'
+    ],
+    forks: [{
+      target: '/postcode',
+      condition: {
+        field: 'country',
+        value: 'United Kingdom'
+      }
+    }],
+    continueOnEdit: true,
+    next: '/address-outside'
+  },
+  '/postcode': {
+    controller: require('./controllers/postcode'),
+    fields: [
+      'postcode'
+    ],
+    forks: [{
+      target: '/address-inside',
+      condition: function checkNI(req) {
+        return _.startsWith(req.form.values.postcode, 'BT');
+      }
+    }, {
+      target: '/address',
+      condition(req) {
+        const addresses = req.sessionModel.get('addresses');
+        return addresses && addresses.length;
+      }
+    }],
+    continueOnEdit: true,
+    next: '/no-postcode'
+  },
+  '/address': {
+    controller: require('./controllers/address'),
+    fields: [
+      'address'
+    ],
+    continueOnEdit: true,
+    next: '/confirm'
+  },
+  '/no-postcode': {
+    controller: require('./controllers/no-postcode'),
+    template: 'no-postcode',
```

Comment:
You don't need to specify the template name if it's the same as the route i.e the route and template are both called 'no-postcode'